### DISCUSSION TITLE
How can I view the public inactive/cancelled sponsorships of a particular GitHub org?

### DISCUSSION THREAD

**Discussion by SonNg2k** [community/community#27712] [Other Feature Feedback, Questions, & Ideas]
When I visit the "Sponsoring" page of mega tech giants on GitHub (take the [Microsoft Sponsoring](https://github.com/orgs/microsoft/sponsoring) and [AWS Sponsoring](https://github.com/orgs/aws/sponsoring) pages for example), I want to view a list of GitHub organizations and developers that **were** financed by these companies in the past. At the moment, GitHub only shows the **active ongoing** sponsorships of these tech giants and hide all of their past sponsorships that were active for a period of time in the past and then cancelled.

Please add this new feature, it's extremely useful because it helps me make a better decision to choose the reliable open-source libs and frameworks for my critical enterprise projects (reliable means they are or were sponsored by tech giants). Thank you!

**Comment by cheshire137** [comment]
Hi there, thanks for your post! I'm a developer on the Sponsors team. We're working now on giving better recognition to past sponsors and acknowledging sponsorships that have expired, both recurring and one-time. That should include giving credit on the sponsor's Sponsoring tab for the maintainers they have sponsored in the past, which would let you see which maintainers, and hopefully through that which projects, the sponsor valued.
